pytta/_plot.py

```diff
@@ -76,8 +76,13 @@ def time(sigObjs, xLabel, yLabel, yLim, xLim, title, decimalSep, timeUnit):
         matplotlib.figure.Figure object.
     """
     if decimalSep == ',':
-        locale.setlocale(locale.LC_NUMERIC, 'pt_BR.UTF-8')
-        plt.rcParams['axes.formatter.use_locale'] = True
+        try:
+            locale.setlocale(locale.LC_NUMERIC, 'pt_BR.UTF-8')
+            plt.rcParams['axes.formatter.use_locale'] = True
+        except:
```

Comment:
Maybe instead of just catching any error, try to inform the user something abaout the actual error. Something like:

try:
    ...
except BaseError as err:
    ...
    print("Error using ',' as decimal separator: {0}.\nSwitching to '.' instead.".format(err))

Reply:
The very best solution would find a way of using ","